tests/common/test_library.py

```diff
@@ -166,6 +166,7 @@ async def test_fetch_orders_filter_errors_one_404(plugin, create_resolver, caplo
     ('Gremlins, Inc.', 'bundle', ['Here, there', 'Gremlins, I'], False),  # blacklisted
     ('Gremlins, Inc.', 'bundle', ['Here, there', 'Gremlins, no match'], True),
     ('Alpha Protocol, Company of Heroes, Rome: Total War, Hell Yeah! Wrath of the Dead Rabbit', 'bundle', [], True),
+    ('Star Wars™ Battlefront™ II (Classic, 2005)', 'bundle', ['STAR WARS™ Battlefront™ II (Classic, 2005)'], False), # Casing in name changed
```

Comment:
```suggestion
    ('Star Wars™ Battlefront™ II (Classic, 2005)', 'bundle', ['STAR WARS™ Battlefront™ II (Classic, 2005)'], False),
```

commit message will be give more light to this change

Reply:
I'm not sure i understand, what's wrong with the comment?
I put that there as it explains why the casing is different, as opposed to every other test that doesn't change it.
Putting it in a commit message makes it a step removed from the code, and requires an extra lookup that may or may not happen if someone decides to "fix it."

Reply:
this comment is meaningful for you, but not for someone who does not know the whole context. To understand why this was added the reader would have to read commit message/PR description anyways.

Better to write something general like: "case sensitivity" and from the test it will be clear that blacklist is case-insensitive (so it includes the case about name changed).